cli/main.c

```diff
@@ -1180,11 +1183,13 @@ static int print_fw_part_info(struct switchtec_dev *dev)
 		bootloader_ro = 0;
 
 	printf("Active Partition:\n");
-	printf("  BOOT \tVersion: %-8s\tCRC: %08lx   %s\n",
-	       bootloader_ver, (long)bootloader.image_crc,
+	printf("  BOOT  \tVersion: %-8s\tCRC: %08lx   %s\n",
+	       bootloader_ver, (long)bootloader.img_crc,
 	       bootloader_ro ? "(RO)" : "");
-	printf("  MAP \tVersion: %-8s\tCRC: %08lx\n",
```

Comment:
why the map related info is removed?

Reply:
my code is based on microsemi/devel, so not include Map paritition information;
now add the Map partition information support;